src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexPersistence.cs

```diff
@@ -205,9 +205,9 @@ public override void Clean(IndexCleanup mode)
             _converter?.Clean();
             if (mode.HasFlag(IndexCleanup.Readers))
             {
-                foreach (LowLevelTransaction llt in _index._indexStorage.Environment().ActiveTransactions.Enumerate())
```

Comment:
They are likely sharing the same `LuceneIndexState`, hence same `IndexSearcher` instance, anyway. Instead of recreating the same searcher we'll recreate just the recent once so the old searcher and its reader will get disposed (via `IndexSearcherDisposer`)